MaterialDesignThemes.Wpf/PopupBox.cs

```diff
@@ -103,6 +103,12 @@ public class PopupBox : ContentControl
         public const string PopupContentControlPartName = "PART_PopupContentControl";
         public const string PopupIsOpenStateName = "IsOpen";
         public const string PopupIsClosedStateName = "IsClosed";
+
+        /// <summary>
+        /// Routed command to be used inside dialog content to close a dialog. Use a <see cref="Button.CommandParameter"/> to indicate the result of the parameter.
```

Comment:
This looks like the old comment from the DialogHost. I would update this to be "Routed command to be used inside of a popup box to close the popup box"